guidelines/requests.md

```diff
@@ -10,6 +10,8 @@ The API accepts only `HTTP POST` requests with `Content-Type` set to `applicatio
 * **Resource** - Resource or domain entity which is the target of the action, always pluralized \(e.g. bills, reservations\)
 * **Action** - Name of the action to be performed on the resource \(e.g. getAll, add, delete\)
 
+> The requested address must exactly match the format specified in the documentation. It is case-sensitive and cannot include a trailing slash or any additional characters. If the address does not match, the server will respond with a` 404 Not Found` error.
```

Comment:
nitpick: I don't think this should be a block quote, but instead I'd highlight the important part:

```suggestion
The operation address **must match the format specified in the documentation**. It is case-sensitive and cannot include a trailing slash or any additional characters. If the address does not match, the server will respond with `404 Not Found` error.
```